src/PKSim.Presentation/Presenters/Main/MenuAndToolBarPresenter.cs

```diff
@@ -276,18 +276,21 @@ private void disableSimulationItems()
          updateSimulationItemsAccordingToSimulationState();
       }
 
-      private void updateSimulationItemsAccordingToSimulationState(bool isRunning = false)
-      {
-         bool enabled = _simulationState.IsActivated && _enabled;
-         bool resultsEnabled = enabled && _simulationState.HasResult;
-         bool enablePopSimulationItems = enabled && !_simulationState.IsIndividual;
-         bool enableIndividualSimulationItems = enabled && _simulationState.IsIndividual;
-         bool enabledPKSimSimulationOnlyItems = enabled && !_simulationState.IsImported;
-
-         var isSimulationAlreadyRunning = (isRunning || _simulationState.IsRunning);
-         _menuBarItemRepository[MenuBarItemIds.Run].Enabled = enabled && !isSimulationAlreadyRunning;
-         _menuBarItemRepository[MenuBarItemIds.RunWithSettings].Enabled = enabled && !isSimulationAlreadyRunning; ;
-
+      /// <summary>
+      ///    Updates the state of all menu items related to simulation according to the current state of the simulation
+      /// </summary>
+      /// <param name="isSimulationRunning">If defined, super seeds the simulationState </param>
+      private void updateSimulationItemsAccordingToSimulationState(bool? isSimulationRunning = null)
+      {
+         var enabled = _simulationState.IsActivated && _enabled;
+         var resultsEnabled = enabled && _simulationState.HasResult;
+         var enablePopSimulationItems = enabled && !_simulationState.IsIndividual;
+         var enableIndividualSimulationItems = enabled && _simulationState.IsIndividual;
+         var enabledPKSimSimulationOnlyItems = enabled && !_simulationState.IsImported;
+         var isRunning = enabled && (isSimulationRunning ?? _simulationState.IsRunning);
```

Comment:
either we pass a flag and we use it if defined, OTHERWUSE we use the state. We do not combine these two flags because this will almost always be wrong